modules/apps/transfer/keeper/relay_forwarding_test.go

```diff
@@ -612,3 +613,169 @@ Test async ack is properly relayed to middle hop after forwarding transfer compl
 // TODO
 Tiemout during forwarding after middle hop execution reverts properly the state changes
 */
+
+func (suite *KeeperTestSuite) setUpForwardingPaths() (path1, path2 *ibctesting.Path) {
+	path1 = ibctesting.NewTransferPath(suite.chainA, suite.chainB)
+	path2 = ibctesting.NewTransferPath(suite.chainB, suite.chainC)
+	path1.Setup()
+	path2.Setup()
+	return path1, path2
+}
+
+type amountType int
+
+const (
+	escrow amountType = iota
+	balance
+)
+
+func (suite *KeeperTestSuite) assertAmountOnChain(chain *ibctesting.TestChain, balanceType amountType, amount sdkmath.Int, denom string, errorMsg string) {
```

Comment:
This is a nice helper function. We. could probably use it in other tests as well. And maybe we can generate the error message within the function itself, so that it doesn't need to be passed in.

Reply:
Done! Do you have any suggestion on where in ibctesting this aux function could be moved to?

Reply:
Not sure... Maybe one of the other guys have some good idea. :)

Reply:
Would it make sense to just add it as a method to `TestChain` itself? One less parameter to send in as well.

Reply:
I'm a bit hesitant to add this to the `TestChain` and increase the public API for a small helper fn, I think I would be happy enough just having it on the test suite as is.